src/deploy/db/mongo-adapter/get-adapter.ts

```diff
@@ -1,34 +1,38 @@
 import { MongoDBAdapter } from "./mongo-adapter";
 import { getLogger } from "../../logger/create-logger";
-import { mongoDbName, mongoURILocal } from "./constants";
+import { DEFAULT_MONGO_DB_NAME, DEFAULT_MONGO_URI, DEFAULT_MONGO_VERSION } from "./constants";
 
 let mongoAdapter : MongoDBAdapter | null = null;
 
 
 export const getMongoAdapter = async () : Promise<MongoDBAdapter> => {
   const checkParams = {
-    // TODO dep: fix type assertion error here
-    dbUri: process.env.MONGO_DB_URI!,
-    dbName: process.env.MONGO_DB_NAME!,
+    dbUri: process.env.MONGO_DB_URI
+      ? process.env.MONGO_DB_URI
+      : DEFAULT_MONGO_URI,
+    dbName: process.env.MONGO_DB_NAME
+      ? process.env.MONGO_DB_NAME
+      : DEFAULT_MONGO_DB_NAME,
   };
 
   const logger = getLogger();
 
   const params = {
     logger,
-    clientOpts: !!process.env.MONGO_DB_CLIENT_OPTS
+    clientOpts: process.env.MONGO_DB_CLIENT_OPTS
       ? JSON.parse(process.env.MONGO_DB_CLIENT_OPTS)
       : undefined,
-    // TODO dep: add better way to set version ENV var is not the best !
-    version: process.env.MONGO_DB_VERSION,
+    version: process.env.MONGO_DB_VERSION
+      ? process.env.MONGO_DB_VERSION
+      : DEFAULT_MONGO_VERSION,
   };
 
   if (!checkParams.dbUri && !checkParams.dbName) {
```

Comment:
this will never be hit since you have this - https://github.com/zer0-os/zNS/pull/70/files#diff-eb1e69c1e785f8b52956c5da9f661f19f2fc3f309c61c7373dcf448776ec0410R10-R15

Reply:
Will remove this check